### PR TITLE
[iOS] Add Loading for TimetableView

### LINKED ISSUE #648: [iOS] Add Loading for TimetableView
**Idea Description**

- Title says all.
- Currently, empty TimetableView is shown in loading state.

**Reference images and links**

### PR DESCRIPTION
**Pull request by NakaokaRei**
## Issue
- close #648

## Overview (Required)
- added LoadingView
- show LoadingView when loading 

## Screenshot

Before | After
:--: | :--:
Simulator Screen Shot - iPhone 14 Pro - 2022-09-22 at 01 44 09 | Simulator Screen Shot - iPhone 14 Pro - 2022-09-22 at 01 43 42

### CODE REVIEW

File: app-ios/Sources/CommonComponents/LoadingView.swift [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import SwiftUI
+
+public struct LoadingView: View {
+
+    public init() {}
+
+    public var body: some View {
+        ZStack {
+            Color.black
```
Comment (ry-itto) [inline app-ios/Sources/CommonComponents/LoadingView.swift:9]: Please use background color defined in Theme

File: app-ios/Sources/CommonComponents/LoadingView.swift [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import SwiftUI
+
+public struct LoadingView: View {
```
Comment (ry-itto) [inline app-ios/Sources/CommonComponents/LoadingView.swift:3]: This view can be used in `SponsorView` as well.
Please apply this too 🙏🏼

File: app-ios/Sources/CommonComponents/LoadingView.swift (hunk 1 above)
Comment (NakaokaRei) [reply app-ios/Sources/CommonComponents/LoadingView.swift]: I fixed at https://github.com/DroidKaigi/conference-app-2022/pull/656/commits/263759bafe64ae4a91351949769ab98797462cbd !

File: app-ios/Sources/CommonComponents/LoadingView.swift (hunk 2 above)
Comment (NakaokaRei) [reply app-ios/Sources/CommonComponents/LoadingView.swift:4]: I applied `LoadingView` at https://github.com/DroidKaigi/conference-app-2022/pull/656/commits/c7c73014c48543fb89b651526570ff2432fb1620 !

Comment (ry-itto) [reply app-ios/Sources/CommonComponents/LoadingView.swift:4]: I think  `AssetColors.background.swiftUIColor` should be used here, is it not appropriate?

File: app-ios/Sources/SponsorFeature/SponsorView.swift [hunk 3]
```diff
@@ -85,9 +85,7 @@ public struct SponsorView: View {
         WithViewStore(store) { viewStore in
             ZStack(alignment: .center) {
                 if viewStore.isLoading {
-                    ProgressView()
-                        .progressViewStyle(CircularProgressViewStyle(tint: AssetColors.onBackground.swiftUIColor))
```
Comment (ry-itto) [inline app-ios/Sources/SponsorFeature/SponsorView.swift:89]: `.progressViewStyle(CircularProgressViewStyle(tint: AssetColors.onBackground.swiftUIColor))`

This modifier is useful for combining with background color 🙏🏼

Comment (NakaokaRei) [reply app-ios/Sources/SponsorFeature/SponsorView.swift:92]: I fixed `LoadingView` style at https://github.com/DroidKaigi/conference-app-2022/pull/656/commits/15d19ebaac148b557555bc9f3b173aac5e895718 !

File: app-ios/Sources/TimetableFeature/TimetableView.swift [hunk 4]
```diff
@@ -100,18 +107,23 @@ public struct TimetableView: View {
...
-                            .onScroll {
-                                viewStore.send(.scroll($0))
-                            }
-                    } else {
-                        TimetableListView(store: store)
-                            .scrollThreshold(Self.scrollThreshold)
-                            .onScroll {
-                                viewStore.send(.scroll($0))
-                            }
+                    ZStack {
```
[the remark this answers is not in the document]
Comment (NakaokaRei) [reply app-ios/Sources/TimetableFeature/TimetableView.swift]: Certainly not necessary.
I fixed at https://github.com/DroidKaigi/conference-app-2022/pull/656/commits/70fcad1f9c33c588a5154a9b51b13b3524c89da8 !

File: app-ios/Sources/Theme/Resources/Colors.xcassets/black.colorset/Contents.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (ry-itto) [inline app-ios/Sources/Theme/Resources/Colors.xcassets/black.colorset/Contents.json:1]: Can you remove this file?
This is not needed for current situation 🙏🏼

**Review by ry-itto — APPROVED** [review]
Looks good!
Please fix a point 🙏🏼 After fixing, I'll merge this 🙏🏼

File: app-ios/Sources/Theme/Resources/Colors.xcassets/black.colorset/Contents.json (hunk 5 above)
Comment (NakaokaRei) [reply app-ios/Sources/Theme/Resources/Colors.xcassets/black.colorset/Contents.json]: I deleted at https://github.com/DroidKaigi/conference-app-2022/pull/656/commits/bed0feda90979219bb32e1ab140bf9a9dc05aec6 !

**Comment by ry-itto** [comment]
Thanks! I'll merge this🙏

**Pull request merged**